voluptuous_serialize/__init__.py

```diff
@@ -12,12 +12,17 @@
 }
 
 
-def convert(schema):
+def convert(schema, custom_serializer=None):
     """Convert a voluptuous schema to a dictionary."""
     # pylint: disable=too-many-return-statements,too-many-branches
     if isinstance(schema, vol.Schema):
         schema = schema.schema
 
+    if custom_serializer:
+        val = custom_serializer(schema)
+        if val:
```

Comment:
Actually thinking about it now, we should compare it to a sentinel object so we can allow to return `None`.

Add to top of file:

```python
UNSUPPORTED = object()
```

```suggestion
        if val not is UNSUPPORTED:
```